fix(console): app details

eliobischof opened the pull request:
The app details for OIDC apps show the SAML config and URLs. In the OIDC apps URLs section, the OIDC URLs are not even listed, only the useless SAML URLs.

This PR fixes this bug introduced with #7572.

The following screenshots show the new details pages.

# API
![image](https://github.com/zitadel/zitadel/assets/12727842/4833c3d5-9d57-41a6-a503-beb41a2cccd8)
![image](https://github.com/zitadel/zitadel/assets/12727842/f2daf2be-a1fe-4b1b-bb75-7095acecf33d)

# OIDC
![image](https://github.com/zitadel/zitadel/assets/12727842/584f2f12-e39b-408e-aa97-957003c68c44)
![image](https://github.com/zitadel/zitadel/assets/12727842/a4d16cfb-6237-4d54-b7da-1f469decf0ea)

# SAML
![image](https://github.com/zitadel/zitadel/assets/12727842/36afc605-ab5a-472d-b66c-6251f69e9cc5)
![image](https://github.com/zitadel/zitadel/assets/12727842/f595c480-0673-4200-859b-c9d50b384bc3)

### Definition of Ready

- [x] I am happy with the code
- [x] Short description of the feature/issue is added in the pr description
- [x] PR is linked to the corresponding user story
- [x] Acceptance criteria are met
- [x] All open todos and follow ups are defined in a new ticket and justified
- [x] Deviations from the acceptance criteria and design are agreed with the PO and documented.
- [x] No debug or dead code
- [x] My code has no repetitions
- [x] Critical parts are tested automatically
- [ ] Where possible E2E tests are implemented
- [ ] Documentation/examples are up-to-date
- [x] All non-functional requirements are met
- [ ] Functionality of the acceptance criteria is checked manually on the dev system.

livio-a commented:
works, but IMO we should add at least the `introspection_endpoint` and `jwks_uri` and maybe even `revocation_endpoint` for APIs.

eliobischof commented:
> works, but IMO we should add at least the `introspection_endpoint` and `jwks_uri` and maybe even `revocation_endpoint` for APIs.

I mapped the URLs explicitly, fixed their order, added the mentioned URLs to the API and prettified the keys.
![image](https://github.com/zitadel/zitadel/assets/12727842/2fed6d8d-76e1-4a54-9d90-6a50eb988e76)

livio-a commented:
sorry apparently overlooked that before, but just noted that the issuer is missing on the OIDC configs
and wouldn't it make sense to have the entityID / metadata endpoint on the SAML config, resp. could we even use that for the link instead of well-known? maybe also any input on that from @stebenz?
